keeper/msg_server_test.go

```diff
@@ -44,6 +45,168 @@ import (
 
 const ONE = int64(1e6)
 
+func TestUnbalanceAndRebalanceIsNotConservative(t *testing.T) {
+	usdnLiquidity := sdk.NewCoin("uusdn", math.NewInt(int64(1_000_000*ONE)))
+	usdcLiquidity := sdk.NewCoin("uusdc", math.NewInt(int64(1_000_000*ONE)))
```

Comment:
No need to cast to int64 since `ONE` already is
```suggestion
	usdnLiquidity := sdk.NewCoin("uusdn", math.NewInt(1_000_000*ONE))
	usdcLiquidity := sdk.NewCoin("uusdc", math.NewInt(1_000_000*ONE))
```